resources/definitions/os_discovery/televes-cs.yaml

```diff
@@ -0,0 +1,99 @@
+mib: TREDESS-DVBT-MIB
+
+modules:
+  os:
+    sysName: TREDESS-DVBT-MIB::sysNameDBTM.0
+    sysName_replace:
+      - '/_+/'
+    sysName_fallback: TREDESS-DVBT-MIB::sysNameDBTM.0
+    sysDescr: TREDESS-DVBT-MIB::sysDescrDBTM.0
+    location: TREDESS-DVBT-MIB::sysNameDBTM.0
+    location_replace:
+      - '/_+/'
+    version: TREDESS-DVBT-MIB::sysManagementVersionDBTM.0
+  sensors:
+    frequency:
+      data:
+        - 
+          oid: TREDESS-DVBT-MIB::muxFreqIn
+          index: muxFreqIn.{{ $index }}
+          num_oid: '.1.3.6.1.4.1.23180.2.1.1.1.3.1.1.1.6.{{ $index }}'
+          descr: 'Slot {{ $index }}'
+          group: Frequency In
+        - 
+          oid: TREDESS-DVBT-MIB::muxFreqOut
+          index: muxFreqOut.{{ $index }}
+          num_oid: '.1.3.6.1.4.1.23180.2.1.1.1.3.1.1.1.7.{{ $index }}'
+          descr: 'Slot {{ $index }}'
+          group: Frequency Out
+    count:  
+      data:
+        - 
+          oid: TREDESS-DVBT-MIB::muxChannelIn
+          index: muxChannelIn.{{ $index }}
+          num_oid: '.1.3.6.1.4.1.23180.2.1.1.1.3.1.1.1.4.{{ $index }}'
+          group: Channel In
+          descr: 'Slot {{ $index }}'
+        - 
+          oid: TREDESS-DVBT-MIB::muxChannelOut
+          index: muxChannelOut.{{ $index }}
+          num_oid: '.1.3.6.1.4.1.23180.2.1.1.1.3.1.1.1.5.{{ $index }}'
+          group: Channel Out
+          descr: 'Slot {{ $index }}'
+        - 
+          oid: muxMeasurementTable
```

Comment:
Can you update oid, value and anywhere else an oid is called with the `MIB-NAME::OID` format please.